src/test/index.ts

```diff
@@ -30,3 +28,73 @@ describe('Hello Query', () => {
     });
   });
 });
+const input = {
+  name: 'UserTeste1',
+  email: 'userteste1@email.com',
+  password: '1234abc',
+  birthDate: '10-10-2000'
+};
+const input2 = {
+  name: 'UserTeste2',
+  email: 'userteste2@email.com',
+  password: '1234abc',
+  birthDate: '10-10-2000'
+};
+describe('CreateUser Mutation', async () => {
+  beforeEach(async () => {
+    await AppDataSource.getRepository(User).delete({
+      email: 'userteste1@email.com'
+    });
+  });
```

Comment:
Pra gente poder manter uma independência maior entre os testes, a gente recomenda limpar o banco entre cada caso de teste. Se precisar de alguma coisa no banco antes do caso de teste, como é o seu teste de e-mail repetido, por exemplo, vc pode fazer um setup pra isso dentro do it desse teste.